### PR TITLE
Feature/wizard analytics page trng 1022

### PR DESCRIPTION
**Pull request by SimonTheSourcerer**

### CODE REVIEW

File: Editor/UI/Wizard/FirstTime/AnalyticsPage.cs [hunk 1]
```diff
@@ -1,17 +1,76 @@
...
+            {
+                AnalyticsUtils.SetTrackingTo(AnalyticsState.Enabled);
+            }
+        }
+
         public override void Draw(Rect window)
         {
+            SetupStyles();
+
+            GUILayout.BeginArea(new Rect(window.x, window.y, window.width, window.height));
```
Comment (aleksei-korolev) [inline Editor/UI/Wizard/FirstTime/AnalyticsPage.cs:33]: Run all titles through https://capitalizemytitle.com/

File: Editor/UI/Wizard/FirstTime/AnalyticsPage.cs [hunk 2]
```diff
@@ -1,17 +1,76 @@
...
+                GUILayout.Label("Help us! Contribute to Improvements!", titleStyle);
+                GUILayout.Box("Innoactive Creator is actively evolving. Please, help us with your feedback.  Provide us your <b>anonymous</b> usage data and contribute to improvements.", textStyle);
```
Comment (aleksei-korolev) [inline Editor/UI/Wizard/FirstTime/AnalyticsPage.cs:35]:
```suggestion
                GUILayout.Box("Innoactive Creator is actively evolving. Please, help us with your feedback. Provide us your <b>anonymous</b> usage data and contribute to improvements.", textStyle);
```

File: Editor/UI/Wizard/FirstTime/AnalyticsPage.cs [hunk 3]
```diff
@@ -1,17 +1,76 @@
...
+
+                GUILayout.Label("We Respect your Privacy", titleStyle);
+                GUILayout.Box("We DO NOT collect any sensitive information such as source code, file names or your courses' structure.\n\nHere is what we collect:\n- exact version of Innoactive Creator\n- exact version of Unity Editor\n- your system's language\n- information about usage of the Innoactive Creator's components\n\nIn order to collect the information above, we store a unique identifier within Unity's Editor Preferences. Your data is anonymized.", textStyle);
```
Comment (aleksei-korolev) [inline Editor/UI/Wizard/FirstTime/AnalyticsPage.cs:38]:
```suggestion
                GUILayout.Box("We DO NOT collect any sensitive information such as the source code of your project, file names, or your courses' structure.\n\nHere is what we collect:\n- the exact version of Innoactive Creator\n- the exact version of Unity Editor\n- your system's language\n- the information about usage of the Creator's components developed by Innoactive.\n\nIn order to collect the information above, we store a unique identifier within Unity's Editor Preferences. Your data is anonymized.", textStyle);
```

File: Editor/UI/Wizard/FirstTime/AnalyticsPage.cs [hunk 4]
```diff
@@ -1,17 +1,76 @@
...
+
+                GUILayout.Space(4f);
+                if (GUILayout.Button(" > Data Privacy Information", urlStyle, GUILayout.ExpandWidth(false)))
+                {
+                    AnalyticsUtils.ShowDataPrivacyStatement();
+                }
+            GUILayout.EndArea();
+        }
+
+        private void SetupStyles()
```
Comment (tomwim) [inline Editor/UI/Wizard/FirstTime/AnalyticsPage.cs:51]: I like the idea of having styles for the wizard (or UI in general) but then in a CreatorEditorStyles helper class. I also would prefer removing the "padding" and solving this by indenting through GUILayout. In my opinion it is clearer and gives more power to the developer

File: Editor/UI/Wizard/FirstTime/AnalyticsPage.cs [hunk 5]
```diff
@@ -1,17 +1,76 @@
...
+                GUILayout.Box("Innoactive Creator is actively evolving. Please, help us with your feedback.  Provide us your <b>anonymous</b> usage data and contribute to improvements.", textStyle);
+
+                GUILayout.Label("We Respect your Privacy", titleStyle);
+                GUILayout.Box("We DO NOT collect any sensitive information such as source code, file names or your courses' structure.\n\nHere is what we collect:\n- exact version of Innoactive Creator\n- exact version of Unity Editor\n- your system's language\n- information about usage of the Innoactive Creator's components\n\nIn order to collect the information above, we store a unique identifier within Unity's Editor Preferences. Your data is anonymized.", textStyle);
+
+                GUILayout.Label("We are Transparent", titleStyle);
+                GUILayout.Box("The Innoactive Creator is open-source. Feel free to check our analytics code in <b>Core/Editor/Analytics</b>\n\nIf you want to opt-out of tracking, open <b>Innoactive > Creator > Windows > Analytics Settings</b> in the Unity's menu bar and choose <i>disabled</i> from the drop-down menu.", textStyle);
+
+                GUILayout.Space(4f);
```
Comment (tomwim) [inline Editor/UI/Wizard/FirstTime/AnalyticsPage.cs:43]: I would prefer standard vertical and horizontal spaces to remove the usage of magic numbers.

**Pull request merged**